cfme/tests/control/test_bugs.py

```diff
@@ -147,6 +155,48 @@ def hardware_reconfigured_alert(alert_collection):
     alert.delete()
 
 
+@pytest.fixture
+def setup_disk_usage_alert(alert_collection, alert_profile_collection, appliance):
+    if appliance.version < "5.10":
+        pytest.skip("The fix for BZ 1658670 has not been backported to CFME 5.9")
+    # get the current time
+    timestamp = datetime.now()
+    # setup the DB query
+    table = appliance.db.client['miq_alert_statuses']
+    query = appliance.db.client.session.query(table.description, table.evaluated_on)
+    # configure the advanced settings and place a large file on the appliance
+    # disk usage above 1 % will now trigger a disk_usage event
+    appliance.update_advanced_settings(
+        {"server": {"events": {"disk_usage_gt_percent": 1}}}
+    )
+    # create a 1 GB file on /var/www/miq/vmdb/log
+    appliance.ssh_client.run_command(
+        "dd if=/dev/zero of=/var/www/miq/vmdb/log/delete_me.txt count=1024 bs=1048576"
+    )
+    # setup the alert for firing
+    expression = {"expression": "fill_count(Server.EVM Workers, >, 0)"}
+    alert = alert_collection.create(
+        fauxfactory.gen_alpha(),
+        based_on='Server',
+        evaluate=("Expression (Custom)", expression),
+        driving_event="Appliance Operation: Server High /var/www/miq/vmdb/log Disk Usage",
+        notification_frequency="1 Minute",
+    )
+    alert_profile = alert_profile_collection.create(
+        ServerAlertProfile,
+        "Alert profile for {}".format(alert.description),
+        alerts=[alert]
+    )
+    alert_profile.assign_to("Selected Servers", selections=["Servers", "EVM"])
+    yield alert, timestamp, query
+    alert_profile.delete()
+    alert.delete()
+    appliance.update_advanced_settings(
+        {"server": {"events": {"disk_usage_gt_percent": "<<reset>>"}}}
+    )
+    appliance.ssh_client.run_command("rm /var/www/miq/vmdb/log/delete_me.txt")
```

Comment:
**Optional** I think it be helpful to check the RC here, verify the file was indeed deleted.